dashboard/db/migrate/20200928224602_create_join_table_lesson_resource.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableLessonResource < ActiveRecord::Migration[5.0]
+  def change
+    create_join_table :lessons, :resources do |t|
+      # t.index [:lesson_id, :resource_id]
```

Comment:
Why comment out the index creation?

Reply:
It did that automatically for some reason and I didn't catch it. I'll uncomment.